plantcv/__init__.py

```diff
@@ -68,6 +68,7 @@
 from plantcv.gaussian_blur import gaussian_blur
 from plantcv.cluster_contours import cluster_contours
 from plantcv.cluster_contour_splitimg import cluster_contour_splitimg
+from plantcv.rotate_img2 import rotate
```

Comment:
What about naming the file `rotate.py` since the original was `rotate_img.py`, then no need for the version 2 in the filename?